File: solutions/java/Problem6.java
import java.util.Scanner;
public class Problem6 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt(), a=0, b=1;
        for(int i=0; i<n; i++) {
            int temp = a;
            a = b;
            b += temp;
        }
        System.out.println(a);
    }
}

File: solutions/cpp/problem6.cpp
#include <iostream>
using namespace std;
int main() {
    int n, a=0, b=1;
    cin >> n;
    for(int i=0; i<n; i++) {
        int temp = a;
        a = b;
        b += temp;
    }
    cout << a;
    return 0;
}